packages/client/src/pages/SignIn/index.js

```diff
@@ -2,13 +2,27 @@ import React, { useState } from 'react';
 import Text from '../../components/TextBox';
 import Button from '../../components/Button';
 
-import useAuth from '../../providers/AuthProvider'
+import useAuth  from '../../providers/AuthProvider';
+import { Mentor, Mentee } from '../../models';
+
 
 const SignInPage = () => {
-    const { user, signin, signout } = useAuth();
     const [ email, setEmail ] = useState('');
     const [ password, setPassword ] = useState('');
 
+    const { user, signin, signout, signup } = useAuth();
+
+    const handleSignUp = () => {
```

Comment:
I assume this test code is more of a codepointer than anything? Is there any use to it?

Reply:
No use. It will be removed before this gets pushed to master.

Reply:
Kk